zebrad/src/components/mempool/downloads.rs

```diff
@@ -370,7 +382,11 @@ where
             ?MAX_INBOUND_CONCURRENCY,
             "queued transaction hash for download"
         );
-        metrics::gauge!("gossip.queued.transaction.count", self.pending.len() as _);
+        metrics::gauge!(
+            "mempool.currently.queued.transactions.total",
```

Comment:
Naming Nitpick: I think we can just use `transactions` as the unit here:
https://prometheus.io/docs/practices/naming/#metric-names

This change also helps distinguish gauges from counters. (Prometheus or grafana might synthesise a `mempool_rejected_transaction_ids_total` counter anyway. I'm not sure exactly when that happens.)


```suggestion
            "mempool.currently.queued.transactions",
```

Reply:
Done in 660fb787